powerline_git.lua

```diff
@@ -140,4 +139,4 @@ local function addAddonSegment()
 end 
 
 -- Register this addon with Clink
-clink.prompt.register_filter(addAddonSegment, 60)
\ No newline at end of file
+clink.prompt.register_filter(addAddonSegment, delayGit)
```

Comment:
@sp3c1 what is delayGit? and delayNpm?

Reply:
variable in config - forgot to update the sample one
:+1: